src/main/java/net/mcreator/workspace/WorkspaceFolderManager.java

```diff
@@ -173,11 +169,6 @@ public File getWorkspaceBackupsCacheDir() {
 	}
 
 	public boolean isFileInWorkspace(File file) {
-		if (FileIO.isFileSomewhereInDirectory(file, workspaceFolder)) {

```

Comment:
Keep this, if file outside workspace is accessed this is either bug or security breach attempt by a plugin

Reply:
SomeoneElse reported this logging files in external libraries being opened in built-in code viewer, which is not
>either bug or security breach attempt

So there should either be an option to check file quietly or an approach like this.

Reply:
In this case, add silent flag, make default overload with silent false and call silent true for this particular case

Reply:
getPathInWorkspace should call silent, for example, as it has own handling already